coronavirus.md

```diff
@@ -2,17 +2,18 @@
 
 Information for Farset Labs members and visitors during the Coronavirus pandemic, in line with [government guidelines](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/our-approach-to-decision-making-summary-120520.pdf).
 
-### Status: 'Step 2'
-**The space is open to members that went through COVID-19 onboarding, but not visitors or public events, and all events are remote.**
+#### **Status: Step 3**
+
+**The space is open to members that went through COVID-19 onboarding and guests under their supervision, but not public events, and all hackerspace meetings are remote.**
 
 **Coming in to the space is conditional on attending COVID-19 onboarding and agreeing to our [return policy].**
 
-* **❌ Visitors: No**
+* **✅ Visitors: Yes, members can bring a few guests under their supervision.**
```

Comment:
'A few' is concerningly vague; a fixed maximum based on community expressed need is going to be more constructive and easier to handle in the worst case scenario and someone shows up with a dozen people because "there's no definition of a few in the rules"

```suggestion
* **✅ Visitors: Yes, members can bring a up to two guests under their supervision.**
```

Reply:
That's fair, I left it vague intentionally but happy enough if we set a number as well. I'll update it to 4 as we discussed at the Town Hall.